group/templates/group/groupDetail.html

```diff
@@ -45,215 +43,220 @@ <h3 style="display:inline">,&nbsp</h3>
             </div>
           </div>
         </div>
-        <hr>
-        <div class="container-fluid">
-          <div class="Description" style="float:left">
-            <p style="font-size:16px; color:gray; word-wrap:break-word">
-              {{ group_description }}
-            </p>
-          </div>
-        </div>
       </div>
     </div>
   </div>
   <hr>
-  <div class="contest">
-    <div class="panel-group" id="accordion">
-      <div class="panel panel-success">
-        <div class="panel-heading">
-          <h4 class="panel-title">
-            <a data-toggle="collapse" data-parent="#accordion" href="#collapseOne">Running-Contest</a>
-          </h4>
-        </div>
-        <div id="collapseOne" class="panel-collapse collapse in">
-          <div class="panel-body">
-            <table class="table table-hover table-striped">
-              <thead>
-                <tr>
-                  <th>Title</th>
-                  <th>Start Time</th>
-                  <th>End Time</th>
-                </tr>
-              </thead>
-              <tbody>
-                {% for running_contest in rc_list %}
-                  {% if running_contest.is_homework %}
-                    <tr style="background-color:#F1F7EA">
-                      <td>{{ running_contest.cname }}</td>
-                      <td>{{ running_contest.start_time }}</td>
-                      <td>{{ running_contest.end_time }}</td>
-                    </tr>
-                  {% else %}
-                    <tr style="background-color:#E3E8E3">
-                      <td>{{ running_contest.cname }}</td>
-                      <td>{{ running_contest.start_time }}</td>
-                      <td>{{ running_contest.end_time }}</td>
+  <div class="container-fluid">
+    <div class="Description" style="float:left">
+      <p style="font-size:16px; color:gray; word-wrap:break-word">
+        {{ group_description }}
+      </p>
+    </div>
+  </div>
+  <hr>
+  <div class="col-lg-offset-0 col-lg-12">
+    <div class="contest">
+      <div style="margin-bottom:15px">
+        <h4 style="text-decoration:underline;color:gray">Traced Contest</h4>
+      </div>
+      <div class="panel-group" id="accordion">
```

Comment:
You can remove it if it is not used in other place in this version.
But I am still curious about why it is ``accordion``.
![2015-05-14 3 14 55](https://cloud.githubusercontent.com/assets/6128538/7618855/7fc1d6a2-f9e7-11e4-901c-fc8f05b1e91f.png)